test/Microsoft.AspNetCore.JsonPatch.Test/DictionaryAdapterTest.cs

```diff
@@ -18,11 +17,11 @@ public void Add_KeyWhichAlreadyExists_ReplacesExistingValue()
             var nameKey = "Name";
             var dictionary = new Dictionary<string, object>(StringComparer.Ordinal);
             dictionary[nameKey] = "Mike";
-            var dictionaryAdapter = new DictionaryAdapter();
-            var resolver = new Mock<IContractResolver>(MockBehavior.Strict);
+            var dictionaryAdapter = new DictionaryAdapter<string, object>();
```

Comment:
We need some tests for things besides `<string, object>`

Reply:
Working on adding these